platforms/ios/src/TangramMap/TGMarker.mm

```diff
@@ -57,13 +57,24 @@ - (instancetype)initWithMapView:(TGMapViewController *)mapView
     return self;
 }
 
-- (void)setStyling:(NSString *)styling
+- (void)setStylingString:(NSString *)styling
```

Comment:
Mmmmm that just doesn't feel quite right to me, linguistically. I think the slight difference in wording is ok since it's a different syntax anyway. If we really dislike it later we can change it.